safe/impact_functions/bases/classified_rh_classified_ve.py

```diff
@@ -1,52 +0,0 @@
-# coding=utf-8
-
-from safe.definitionsv4.layer_modes import layer_mode_classified
-from safe.definitionsv4.layer_geometry import (
-    layer_geometry_point,
-    layer_geometry_line,
-    layer_geometry_polygon,
-    layer_geometry_raster)
-from safe.common.exceptions import MetadataLayerConstraintError
-from safe.impact_functions.base import ImpactFunction
-from safe.impact_functions.bases.layer_types.classified_raster_hazard import (
-    ClassifiedRasterHazardMixin)
-from safe.impact_functions.bases.layer_types.\
-    classified_vector_exposure import ClassifiedVectorExposureMixin
```

Comment:
For me, I always avoid to use `\`. But, sometimes, the path to import is very long, and force me to use `\`.

It's unused legacy code anyway.